completions/fish/wl-paste.fish

```diff
@@ -0,0 +1,57 @@
+function __wayland_seats --description 'Print all wayland seats'
+    set -l seat
+
+    if type -q weston-info
+        set seat (weston-info 2>/dev/null | sed -n '/wl_seat/{n;s/\s*name: //;p}')
+    else if type -q loginctl
+        set seat (loginctl --no-legend --no-pager list-seats 2>/dev/null)
+    end
+
+    # Default seat if we have no other commands
+    if  test -z "$seat"
+        set seat "seat0"
+    end
+    
+    for i in $seat
+        echo "$i"
+    end
+end
+
+function __wl_paste_types --description 'Print types with context'
+    set -l type (commandline -o)
+
+    for i in (seq (count $type))
+        if [ "$type[$i]" = '--primary' ]; or [ "$type[$i]" = '-p' ]
+            set clip "primary"
+        else if [ "$type[$i]" = '--seat' ]; or [ "$type[$i]" = '-s' ]
```

Comment:
I don't think this will handle complex cases like

`wl-paste -pt` (complete here)

which bash completions try to get right. But maybe it's just not worth supporting that?

Reply:
`wl-paste -pt` seems to actually ignore *all* completions, and triggers the default action where it offers files; Looking through other completion files I'm not sure mixing short options actually works in fish.

Reply:
I'm told this does actually work for other programs, seemingly with no extra code from completions. Try `kill -aq <TAB>` if you're on GNU/Linux.

Reply:
No, that was wrong. Indeed it doesn't work.